package/gluon-web-private-wifi/luasrc/lib/gluon/config-mode/model/admin/privatewifi.lua

```diff
@@ -26,6 +27,25 @@ key:depends(enabled, true)
 key.datatype = "wpakey"
 key.default = uci:get('wireless', primary_iface, "key")
 
+local encryption = s:option(ListValue, "encryption", translate("Encryption"))
+encryption:depends(enabled, true)
+encryption:value("psk2", translate("WPA2"))
+if platform.supports_wpa3() then
+	encryption:value("psk3-mixed", translate("WPA2 / WPA3"))
+	encryption:value("psk3", translate("WPA3"))
+end
+encryption.default = uci:get('wireless', primary_iface, 'encryption') or "psk2"
+
+local mfp = s:option(ListValue, "mfp", translate("Management Frame Protection"))
+mfp:depends(enabled, true)
+mfp:value("0", translate("Disabled"))
+if platform.supports_mfp(uci) then
+	mfp:value("1", translate("Optional"))
+	mfp:value("2", translate("Enabled"))
```

Comment:
Required?

Reply:
This was the naming I've used initially, however MFP is not fully-enabled when It's only optional (and required is not the correct way to express such a condition), that's why I've decided to go with enabled here.

Reply:
Can you rephrase that? I'm not understanding what you're saying.